packages/data/lib/repository/preference_local_repository.dart

```diff
@@ -17,4 +24,36 @@ class PreferencesLocalRepositoryImpl implements PreferencesLocalRepository {
       sharedPreferences.setString(_passwordKey, user.password)
     ]);
   }
+
+  @override
+  Future<bool> saveDate(String date) async {
+    final convertedDateResponse = convertStringToDate(date);
+    final savedDate = sharedPreferences.getString(_dateKey);
+    if (savedDate == null) {
+      await sharedPreferences.setString(
```

Comment:
up to u, but on my opinion, is the more preferred way to save date is to save it DateTime.fromMillisecondsSinceEpoch value